### PR TITLE
SONAR-9566 Issues at org level

### PR DESCRIPTION
**Pull request by gregaubert**

### CODE REVIEW

File: server/sonar-web/src/main/js/apps/organizations/navigation/OrganizationNavigation.js [hunk 1]
```diff
@@ -35,6 +36,7 @@ const ADMIN_PATHS = [
 
 export default class OrganizationNavigation extends React.PureComponent {
   props: {
+    currentUser: { isLoggedIn: boolean, showOnboardingTutorial: true },
```
Comment (stas-vilchik) [inline server/sonar-web/src/main/js/apps/organizations/navigation/OrganizationNavigation.js]: remove `showOnboardingTutorial`

File: tests/src/test/java/org/sonarqube/tests/issue/OrganizationIssuesPageTest.java [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+import org.sonarqube.tests.Category6Suite;
+import org.sonarqube.tests.Tester;
+import org.sonarqube.ws.Organizations;
+import org.sonarqube.ws.WsUsers;
+import util.issue.IssueRule;
+
+import static util.ItUtils.restoreProfile;
+import static util.ItUtils.runProjectAnalysis;
+
+public class OrganizationIssuesPageTest {
```
Comment (stas-vilchik) [inline tests/src/test/java/org/sonarqube/tests/issue/OrganizationIssuesPageTest.java]: add this class to `tests/src/test/java/org/sonarqube/tests/Category6Suite.java`

File: server/sonar-web/src/main/js/apps/organizations/navigation/OrganizationNavigation.js (hunk 1 above)
Comment (gregaubert) [inline server/sonar-web/src/main/js/apps/organizations/navigation/OrganizationNavigation.js]: +1

**Comment by gregaubert** [comment]
Thanks @stas-vilchik, I removed the organization name in breadcrumbs and and the project facet in the last commit.
As for the "My Issues" filter I just discussed with Fabrice because it seemed to weird to me to remove it just here (we have it at project level) and we agreed to keep the "My Issues" at organization level too. I updated the MMF.

**Comment by gregaubert** [comment]
@stas-vilchik Fixed in commit 79659e4, thanks!

**Pull request merged**